src/main/java/com/dsrm/dsrmbackend/services/impl/AuthServiceImpl.java

```diff
@@ -22,10 +22,10 @@
 @RequiredArgsConstructor
 public class AuthServiceImpl implements AuthService {
 
-    @Value("${dsrm.auth.jwtExpirationMs}")
+    @Value("${jwt.ExpirationMs}")
```

Comment:
Why did you change that? Leave it as it is. It is dsrm specific so i believe the prefix dsrm was good idea.
But if you want you can change this to ```dsrm.auth.jwt.expirationMs``` and ```dsrm.auth.jwt.refreshExpirationMs```. It would be slightly better